src/module/entities/helper.js

```diff
@@ -580,18 +581,26 @@ module.exports = class UserProjectsHelper {
 
 					// Prepare registryDetails based on singleEntity data
 					let registryDetails = {}
-					if (singleEntity.locationId) {
-						registryDetails['locationId'] = singleEntity.locationId
-						if (singleEntity.code) {
-							registryDetails['code'] = singleEntity.code
+					if (singleEntity.externalId) {
+						registryDetails['locationId'] = singleEntity.externalId
+						if (singleEntity.externalId) {
+							registryDetails['code'] = singleEntity.externalId
 						}
 
 						registryDetails['lastUpdatedAt'] = new Date()
 					}
+					
+					let childHierarchyPath = [];
```

Comment:
@MallanagoudaB this was not there before?

Reply:
i asked priyanka this thing is required